.devcontainer/post-create.sh

```
@@ -12,13 +12,33 @@ go mod download
+
+echo "==> Migrating seed database..."
+./sippy migrate --database-dsn="$SIPPY_SEED_DATABASE_DSN"
+
+echo "==> Seeding seed database..."
+./sippy seed-data --database-dsn="$SIPPY_SEED_DATABASE_DSN"
+
+echo "==> Migrating prod-like database..."
+./sippy migrate --database-dsn="$SIPPY_PRODLIKE_DATABASE_DSN"
+
+if [ -n "${HOST_WORKSPACE_FOLDER:-}" ]; then
+  host_project_dir=$(echo "$HOST_WORKSPACE_FOLDER" | sed 's|/|-|g')
```

Shellcheck (0.11.0)

[style] See if you can use ${variable//search/replace} instead.

(SC2001)